test/blackbox-tests/test-cases/toplevel-stanza/preprocessors/fooppx.ml

```diff
@@ -0,0 +1,10 @@
+open Ppxlib
+
+let rules =
+  let extension = 
+    Extension.declare "get_c" Expression Ast_pattern.__ (fun ~loc ~path:_ _ ->
```

Comment:
I'd use something else than `get_c`. IIRC, in the original example it was because the library was called `c`. What about `foo` or `test`? It's not super important, but readers might ask themselves what `get_c` means.